VirtualReality/MRML/vtkMRMLVirtualRealityViewNode.cxx

```diff
@@ -331,6 +332,53 @@ bool vtkMRMLVirtualRealityViewNode::SetAndObserveHMDTransformNode(vtkMRMLLinearT
   return true;
 }
 
+//----------------------------------------------------------------------------
+vtkMRMLLinearTransformNode* vtkMRMLVirtualRealityViewNode::GetNthTrackerTransformNode(int index)
+{
+  return vtkMRMLLinearTransformNode::SafeDownCast(
+    this->GetNthNodeReference(this->TrackerTransformRole, index));
+}
+
+//----------------------------------------------------------------------------
+const char* vtkMRMLVirtualRealityViewNode::GetNthTrackerTransformNodeID(int index)
+{
+  return this->GetNthNodeReferenceID(this->TrackerTransformRole, index);
+}
+
+//----------------------------------------------------------------------------
+vtkMRMLLinearTransformNode* vtkMRMLVirtualRealityViewNode::AddAndObserveTrackerTransformNodeID(const char *nodeId)
+{
+  return vtkMRMLLinearTransformNode::SafeDownCast(this->AddAndObserveNodeReferenceID(this->TrackerTransformRole, nodeId));
```

Comment:
It does not have to be a linear transform node. We should allow using a generic vtkMRMLTransformNode, too.